docker-compose.yml

```diff
@@ -7,19 +7,19 @@ proxy:
    - "8888:80"
    - "5555:5555"
   volumes:
-   - proxy/nginx/conf:/opt/openresty/nginx/conf
-   - proxy/nginx/lualib:/opt/openresty/nginx/lualib
-   - proxy/nginx/sites-enabled:/opt/openresty/nginx/sites-enabled
-   - proxy/nginx/sites-available:/opt/openresty/nginx/sites-available
+   - ./proxy/nginx/conf:/opt/openresty/nginx/conf
+   - ./proxy/nginx/lualib:/opt/openresty/nginx/lualib
+   - ./proxy/nginx/sites-enabled:/opt/openresty/nginx/sites-enabled
+   - ./proxy/nginx/sites-available:/opt/openresty/nginx/sites-available
 api:
   extends:
     file: base.yml
     service: api
-  build: api
+  image: jazzdd/alpine-flask
   net: container:proxy
   volumes:
-   - api/ceryx:/ceryx/ceryx
-   - api/bin:/ceryx/bin
+   - ./api:/app
+  command: "-d"
```

Comment:
no argument is missing, there is a entry_point in the docker file,
the argument is -d ( debug mode) because we dont use the nginx mode of the image